### PR TITLE
MLPERF BERT submission scripts

### PR DESCRIPTION
**Pull request by Eliulm**
FYI, [bert_green_08051012_20093.log](https://github.com/user-attachments/files/16509025/bert_green_08051012_20093.log) ran in 4:15min. 

Log is compliant with the checker: 

```
eliulm@tiny14:~$ python -m mlperf_logging.compliance_checker --usage training --ruleset 4.0.0 ./tinygrad/bert_green_08051012_20093.log 
INFO - Running compliance on file: ./tinygrad/bert_green_08051012_20093.log
INFO -  Compliance checks: training_4.0.0/common.yaml
INFO -  Compliance checks: training_4.0.0/closed_common.yaml
INFO -  Compliance checks: training_4.0.0/closed_bert.yaml
** Logging output also at compliance_checker.log
INFO - SUCCESS
```

### CODE REVIEW

File: examples/mlperf/training_submission_v4.1/tinycorp/benchmarks/bert/implementations/tinybox_red/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+This problem uses BERT for NLP.
+
+## Requirements
+
+Install tinygrad and mlperf-logging from master.
+```
+git clone https://github.com/tinygrad/tinygrad.git
+python3 -m pip install -e ".[mlperf]"
+```
+Also install tqdm, tensorflow and pillow.
```
Comment (chenyuxyz) [inline examples/mlperf/training_submission_v4.1/tinycorp/benchmarks/bert/implementations/tinybox_red/README.md:12]: do we need pillow?

File: examples/mlperf/training_submission_v4.1/tinycorp/benchmarks/bert/implementations/tinybox_red/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+```
+BASEDIR="/raid/datasets/wiki" python3 extra/datasets/wikipedia.py pre-eval
+```
+## Running
+
+### tinybox_green
+
+#### Steps to run benchmark
+```
+examples/mlperf/training_submission_v4.0/tinycorp/benchmarks/bert/implementations/tinybox_green/run_and_time.sh
```
Comment (chenyuxyz) [inline examples/mlperf/training_submission_v4.1/tinycorp/benchmarks/bert/implementations/tinybox_red/README.md:70]: v4.1?

File: examples/mlperf/training_submission_v4.1/tinycorp/benchmarks/bert/implementations/tinybox_red/dev_beam.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+export PYTHONPATH="."
+export MODEL="bert"
+export DEFAULT_FLOAT="HALF" GPUS=6 BS=84 EVAL_BS=6
+
+export BEAM=4
+export BASEDIR="/raid/datasets/wiki"
+
+echo "NOTE: DISABLING DROPOUT - UNSET FOR REAL SUBMISSION RUN"
+export DISABLE_DROPOUT=1 # NOTE: Unset flag for real submission run.
```
Comment (chenyuxyz) [inline examples/mlperf/training_submission_v4.1/tinycorp/benchmarks/bert/implementations/tinybox_red/dev_beam.sh:11]: change NOTE to TODO

**Comment by chenyuxyz** [comment]
cool, will try this today

File: examples/mlperf/training_submission_v4.1/tinycorp/benchmarks/bert/implementations/tinybox_red/README.md (hunk 1 above)
Comment (Eliulm) [reply examples/mlperf/training_submission_v4.1/tinycorp/benchmarks/bert/implementations/tinybox_red/README.md:12]: ye, its imported in `dataloader.py` for resnet. Could move the import though

Comment (chenyuxyz) [reply examples/mlperf/training_submission_v4.1/tinycorp/benchmarks/bert/implementations/tinybox_red/README.md:12]: yea move it so we don't need pillow

ideally we also want to use our tqdm but that can be done separately
